internal/gitinterface/commit_test.go

```diff
@@ -388,61 +243,145 @@ func createTestSignedCommit(t *testing.T) *object.Commit {
 	}
 	testCommit.PGPSignature = sig.String()
 
-	return testCommit
+	// Re-encode with the signature
+	commitEncoded = goGitRepo.Storer.NewEncodedObject()
+	if err := testCommit.Encode(commitEncoded); err != nil {
+		t.Fatal(err)
+	}
+
+	commitID, err := goGitRepo.Storer.SetEncodedObject(commitEncoded)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	commitHash, err := NewHash(commitID.String())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return commitHash
 }
 
-func createTestSSHSignedCommits(t *testing.T) []*object.Commit {
+func createTestSigstoreSignedCommit(t *testing.T, repo *Repository) Hash {
 	t.Helper()
 
-	testCommits := []*object.Commit{}
-
-	signingKeys := [][]byte{rsaSSHPrivateKeyBytes, ecdsaSSHPrivateKeyBytes}
-
-	for _, keyBytes := range signingKeys {
-		testCommit := &object.Commit{
-			Author: object.Signature{
-				Name:  testName,
-				Email: testEmail,
-				When:  testClock.Now(),
-			},
-			Committer: object.Signature{
-				Name:  testName,
-				Email: testEmail,
-				When:  testClock.Now(),
-			},
-			Message:  "Test commit",
-			TreeHash: EmptyTree(),
-		}
-
-		commitBytes, err := getCommitBytesWithoutSignature(testCommit)
-		if err != nil {
-			t.Fatal(err)
-		}
-		signer, err := ssh.ParsePrivateKey(keyBytes)
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		sshSig, err := sshsig.Sign(bytes.NewReader(commitBytes), signer, sshsig.HashSHA512, namespaceSSHSignature)
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		sigBytes := sshsig.Armor(sshSig)
-		testCommit.PGPSignature = string(sigBytes)
-
-		testCommits = append(testCommits, testCommit)
-	}
-
-	return testCommits
+	goGitRepo, err := repo.GetGoGitRepository()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	testCommit := &object.Commit{
+		Hash: plumbing.NewHash("d6b230478965e25477263aa65f1ca6d23d0c0d97"),
+		Author: object.Signature{
+			Name:  "Aditya Sirish",
```

Comment:
No, this is an actual commit I signed using gitsign, and there's a corresponding entry in the public good instance of rekor (https://search.sigstore.dev/?logIndex=29674028). Technically, you can verify that against my identity and it will pass, but we don't want to do that live check in a unit test. For now, I'd leave this in, but we'll probably make this more generic when we figure out #73.